src/main/kotlin/lotto/domain/Rank.kt

```diff
@@ -0,0 +1,11 @@
+package lotto.domain
+
+enum class Rank(
+    val reward: Long,
+) {
+    FIRST(2000000000),
+    SECOND(1500000),
+    THIRD(50000),
+    FOURTH(5000),
```

Comment:
```suggestion
    FIRST(2_000_000_000),
    SECOND(1_500_000),
    THIRD(50_000),
    FOURTH(5_000),
```
세자리 단위로 언더바를 붙이면 가독성이 좋아져요

Reply:
수정하였습니다.

**b097365f**